addons/website/static/src/js/utils.js

```diff
@@ -332,7 +348,7 @@ export async function svgToPNG(src) {
                 finalImg.onload = () => {
                     resolve(finalImg);
                 };
-                finalImg.src = `data:image/svg+xml;base64,${svg64}`;
+                finalImg.src = `data:image/${format};base64,${svg64}`;
```

Comment:
> Isn't there more to modify in this function ? .... svgEl and svg64 ...

As the comment mention, `checkImg` is there for a specific bug in Firefox with SVGs. So for now, the code will no go to the svg stuff if it's not an svg. Let me know if it is not clear :)

> In the first case, then maybe something to change to include other formats.

Adding support for other formats is premature IMO

Reply:
Okay but then, why was it needed to use the `format` argument, since this part will only be reached for a particular svg issue in Firefox ? 
Maybe keep ```finalImg.src = `data:image/svg+xml;base64,${svg64}`;``` here and use the `format` in a `if` somewhere above (like here :arrow_down:)
```js
// In case we receive a loaded image and that this image is not problematic,
// we can convert it to PNG directly.
if (src instanceof HTMLImageElement) {
    const loadedImgEl = src;
    if (checkImg(loadedImgEl)) { // <-- here 
        return toPNGViaCanvas(loadedImgEl);
    }
    src = loadedImgEl.src;
}
```
Something like `if (checkImg(...) or format === "webp")` then no problem you can return ? It will be more clear that the rest only concerns the SVGs 

Sorry for nitpicking, what you did works, I just don't find it consistent codewise :slightly_smiling_face:

Reply:
done :heavy_check_mark: